package.json

```diff
@@ -457,6 +523,29 @@
               "dfdlDebugger": {
                 "type": "object",
                 "description": "Configuration for debugger. Settings are logging (level and file)",
+                "properties": {
+                  "logging": {
+                    "type": "object",
+                    "description": "DFDL Debugger Logging Configuration",
+                    "properties": {
+                      "file": {
+                        "type": "string",
+                        "description": "Path to file to save logs at",
+                        "default": "/tmp/daffodil-debugger.log"
```

Comment:
Does Windows have `/tmp`?  I would opt for `${workspaceFolder}/daffodil-debugger.log`, as we do for Ωedit.